services/mailer/mail_issue.go

```diff
@@ -15,15 +15,15 @@ import (
 	"github.com/unknwon/com"
 )
 
-func mailSubject(issue *models.Issue) string {
+func defaultMailSubject(issue *models.Issue) string {
```

Comment:
This doesn't seem to be an actual default as the included template would override it so this text is never used, so this isn't used ever (same for all references below)

Reply:
@mrsdizzie You are right. Is more of a fallback mail subject. I'll change it.

Reply:
It's for cases where the template is so badly crafted that it renders an empty string. Unless we consider this should be an actual possibility.

Reply:
@mrsdizzie renamed `defaultMailSubject()` as `fallbackMailSubject()` to better reflect its purpose.